bank/sql_bank/main.go

```diff
@@ -21,23 +21,23 @@ import (
 	"database/sql"
 	"flag"
 	"fmt"
+	"log"
 	"math/rand"
+	"net/url"
 	"os"
 	"time"
 
 	"github.com/lib/pq"
 
-	"github.com/cockroachdb/cockroach/security"
-	"github.com/cockroachdb/cockroach/security/securitytest"
-	"github.com/cockroachdb/cockroach/server"
-	"github.com/cockroachdb/cockroach/util/log"
+	_ "github.com/cockroachdb/cockroach/sql/driver"
 )
 
+var dbURL = flag.String("db-url", "", "URL to connect to a running database.")
+var dbDriver = flag.String("db-driver", "cockroach", "DB driver to use.")
```

Comment:
the postgres driver is intentionally not mentioned in the `README` as it currently does not work.